### PR TITLE
add jupyterlab notebooks

### PR DESCRIPTION
**Pull request by ebattat**
Improvements:

1. Add jupyterlab notebooks: elasticsearch_operations and analyze_prometheus_logs
2. Add support classes for notebooks: elasticsearch_widgets.py and analyze_prometheus_logs.py
3. Add required packages for setup.py and requirement
4. Add notebooks files to manifest.ini

### CODE REVIEW

File: jupyterlab/templates/elasticsearch_operations/elasticsearch_widgets.py
Comment (RobertKrawitz) [inline jupyterlab/templates/elasticsearch_operations/elasticsearch_widgets.py:66]: If I'm interpreting the Github display correctly, this file needs a trailing newline.

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]
One trivial change (need a trailing newline in elasticsearch_widgets.py).

File: jupyterlab/templates/elasticsearch_operations/elasticsearch_widgets.py
Comment (RobertKrawitz) [reply jupyterlab/templates/elasticsearch_operations/elasticsearch_widgets.py:66]: I would like @athiruma to review this also, as I'm not familiar with this area.

Comment (ebattat) [reply jupyterlab/templates/elasticsearch_operations/elasticsearch_widgets.py:66]: @RobertKrawitz, its a jupyter widgets method, I test on jupyterlab and it works fine

Comment (RobertKrawitz) [reply jupyterlab/templates/elasticsearch_operations/elasticsearch_widgets.py:66]: There's no issue with the code, but Linux (POSIX in general) text files are required to always have a trailing newline, reference https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206  ("A sequence of zero or more non- <newline> characters plus a terminating <newline> character.") and https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3

File: jupyterlab/templates/logs_analyzer/analyze_prometheus_logs.py
Comment (athiruma) [inline jupyterlab/templates/logs_analyzer/analyze_prometheus_logs.py:69]: I think chunk size should be const

File: jupyterlab/templates/elasticsearch_operations/elasticsearch_widgets.py
Comment (ebattat) [reply jupyterlab/templates/elasticsearch_operations/elasticsearch_widgets.py]: @RobertKrawitz, right update it.

**Review by RobertKrawitz — APPROVED** [review]

**Comment by RobertKrawitz** [comment]
I've approved it.  I'm not marking it approved pending @athiruma approval.

File: jupyterlab/templates/logs_analyzer/analyze_prometheus_logs.py
Comment (ebattat) [reply jupyterlab/templates/logs_analyzer/analyze_prometheus_logs.py:69]: @athiruma, good catch, fixed it

File: jupyterlab/templates/logs_analyzer/analyze_prometheus_logs.py
Comment (athiruma) [inline jupyterlab/templates/logs_analyzer/analyze_prometheus_logs.py:91]: Zeros are repeated. Can we make it as const?

File: jupyterlab/templates/logs_analyzer/analyze_prometheus_logs.py
Comment (athiruma) [inline jupyterlab/templates/logs_analyzer/analyze_prometheus_logs.py:131]: I think, It is better to specify the input parameters format.

File: jupyterlab/templates/logs_analyzer/analyze_prometheus_logs.py
Comment (ebattat) [reply jupyterlab/templates/logs_analyzer/analyze_prometheus_logs.py:91]: Yes

**Review by athiruma — APPROVED** [review]
/lgtm

**Pull request merged**
